tests/test_api_lib_policy.py

```diff
@@ -3342,6 +3343,30 @@ def test_60_increase_failcounter_on_challenge(self):
         # delete policy
         delete_policy("increase_failcounter_on_challenge")
 
+    def test_61_required_description_for_specified_token_types(self):
+        builder = EnvironBuilder(method='POST',
+                                 data={'user': "hans", "pass": "123456"},
+                                 headers={})
+        env = builder.get_environ()
+        req = Request(env)
+        req.User = User()
+
+        # Check that parameter is not set
+        req.all_data = {"type": "test_type"}
+        self.assertNotIn("require_description", req.all_data)
```

Comment:
this test won't work if the `require_description` key is not added to `all_data` (see above).
We could create a test environment where the `description` parameter is missing just for the lib-tests.

Reply:
fixed in 2ca66d18f3cc56625edbacac143bb58cb9a3ee07.